rust/src/http2/range.rs

```diff
@@ -0,0 +1,241 @@
+/* Copyright (C) 2021 Open Information Security Foundation
```

Comment:
Is the parsing of the content-range string the same for http1 and http2? If so I'd like to consolidate the parsing in Rust

Reply:
whoops already done it seems, please ignore